pages/osx/sips.md

```diff
@@ -0,0 +1,28 @@
+# sips
+
+> Apple Scriptable Image Processing System.
+> Raster/Query images and ColorSync ICC Profiles.
+
+- Specify an output directory so originals does not get modified:
+
+`sips --out /path/to/out ...`
+
+- Resample image at specified size, Image aspect ratio may be altered:
```

Comment:
Should it be resample or resize ?

Reply:
this [page](http://www.richmediacs.com/user_manuals/RMCS_PS_Training/ResizeVsResample.html) explains the difference between resizing and resampling well.

The tldr, is when resizing you're redistributing the amount of pixels. Compared to when you're resampling you're adding or subtracting the amount of pixels in the image.